source/architecture.rst

```diff
@@ -1,230 +1,245 @@
-database. The data is converted into an storage structure and is sent to the corresponding database.
+For internal storage, the MongoDB non-relational database is being used, it
+allows a Sharded Cluster configuration that may be required according to the
+use case.
 
-For internal storage, the MongoDB non-relational database is being used, it allows a Sharded Cluster configuration that
-may be required according to the use case.
-
-The data may also be directed to databases that are external do the *dojot* platform, requiring only a proper
-configuration of Logstash and the data model to be used.
+The data may also be directed to databases that are external do the *dojot*
+platform, requiring only a proper configuration of Logstash and the data model
+to be used.
 
 Logging and Auditing Service
 ****************************
 
-All the services that are part of the *dojot* platform generate usage metrics of its resources and make then available
-to the logging and auditing service, which process this registers and summarize then based on users and applications.
+All the services that are part of the *dojot* platform generate usage metrics
+of its resources and make then available to the logging and auditing service,
+which process this registers and summarize then based on users and
+applications.
 
-The consolidated data is presented back to the services, allowing then, for example, to expose this data to the user via
-a graphical interface, to limit the usage of the system based on resource consumption and quotas associated with users
-or even to be used by billing services to charge users for the utilization of the platform.
+The consolidated data is presented back to the services, allowing then, for
+example, to expose this data to the user via a graphical interface, to limit
+the usage of the system based on resource consumption and quotas associated
+with users or even to be used by billing services to charge users for the
+utilization of the platform.
 
 Kong API Gateway
 ****************
 
-The Kong API Gateways is used as the entry point for applications and external services to reach the services that are
-internal to the dojot platform, resulting in multiple advantages like, for instance, single access point and ease when
-applying rules over the API calls like traffic rate limitation and access control.
+The Kong API Gateways is used as the entry point for applications and external
+services to reach the services that are internal to the dojot platform,
+resulting in multiple advantages like, for instance, single access point and
+ease when applying rules over the API calls like traffic rate limitation and
+access control.
 
 Management Application
 **********************
 
-Web Application responsible for providing responsive interfaces to manage the *dojot* platform, including
-functionalities like:
+Web Application responsible for providing responsive interfaces to manage the
```

Comment:
In other places of this document it was called GUI, it should be standardized

Reply:
Ok